### PR TITLE
Various 60 FPS Fixes

### PR DESCRIPTION
**Pull request by Derpduck**
60 FPS Fixes

Respawn time when switching teams being halved on 60 FPS
Mine flash animation speed
Blade projectile sprite animation speed
Bodies fading faster on 60 FPS
Sandwich cooldown being set incorrectly on client side on 60 FPS
Cloak flicker cooldown ending too fast
Kill-Log clearing too fast
Initial alarms for Rocket moved to create event code instead of using Game Maker's code blocks in the editor
Above alarms are fixed for 60 FPS (if they even do anything anymore)
Rockets use dealFlicker script instead of manually doing the same thing in the collision and explosion events
Medic's first health regeneration starting in half the time (would never matter unless you got hit within 1 second of spawning)
Time before being able to fire weapon upon spawning being halved on 60 FPS
Reflected flare lifetime being halved
Reflected rocket alarms
Flare refire time being half
~~Minigun ammo refill delay ending too fast~~
Sniper using dealFlicker script
Mine explosion using dealFlicker script
Afterburn decay delay being too fast
Impact effect alarms use code instead of Game Maker's code blockss, and fixed for 60 FPS
Bubble pop effect alarms use code instead of Game Maker's code blocks, and fixed for 60 FPS
Notice Object alarms use code instead of Game Maker's code blocks, and fixed for 60 FPS

### CODE REVIEW

**Comment by Medo42** [comment]
Just in time for the "big" update, thanks :)

File: Source/gg2/Objects/InGameElements/DeadGuy.events/Alarm 0.xml [hunk 1]
```diff
@@ -15,7 +15,7 @@
       <arguments>
         <argument kind="STRING">{
     alarm[0]=1;
-    image_alpha -= 1/90;
+    image_alpha -= 1/90 / global.delta_factor;
```
Comment (Medo42) [inline Source/gg2/Objects/InGameElements/DeadGuy.events/Alarm 0.xml]: Seems like this should be a mult

File: Source/gg2/Objects/Weapons/Minigun.events/Begin Step.xml [hunk 2]
```diff
@@ -18,7 +18,8 @@
 else if (ammoCount &lt;= maxAmmo and isRefilling)
     ammoCount += 1 * global.delta_factor;
 if(!readyToShoot and alarm[5] &lt; 25 and !isRefilling)
-    alarm[5] += 1;
+    if(global.run_virtual_ticks)
+        alarm[5] += 1;
```
Comment (Medo42) [inline Source/gg2/Objects/Weapons/Minigun.events/Begin Step.xml]: Probably wrong, should be +=2 or just no change at all

**Comment by Derpduck** [comment]
> There also seem to be more places where alarms are set and not adjusted, we may want to have a look later with grep.

I'm pretty sure the assist time expiration alarms need fixing too if anyone wants to do that.

File: Source/gg2/Objects/Weapons/Minigun.events/Begin Step.xml (hunk 2 above)
Comment (Derpduck) [inline Source/gg2/Objects/Weapons/Minigun.events/Begin Step.xml]: Looking into this more it doesn't seem to do what I initially thought lol so will just revert it.

**Comment by Medo42** [comment]
Merged into my local repo, accidentally onto wareya's changes which I have yet to review. Consider it merged though ;)

I added a new issue (https://github.com/Gang-Garrison-2/Gang-Garrison-2/issues/181) with some more places where timers are used without scaling, but I think tackling this is too much for the April 1st release, and it should go into one or more different PRs anyway.

**Pull request closed without merging**